aws/studio_container_setup.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
Can you modify this notebook so it's more general on how to use a custom container for studio? You can mention steps specifically for the HPO example but let's remove all the extra detail from the SM HPO notebook

Reply:
yes! I'll follow the article you linked from above.

this notebook was more so that someone could essentially copy the normal HPO_extended example in Studio, i.e. a "how to pull, build, and push a container to the ECR" tutorial, for the purposes of passing the ECR URI into an Estimator. so that's more what the "container setup" refers to - setting everything up to be able to run the HPO example in Studio. the article's tutorial would be more of a tutorial for how to create a custom container to use as the kernel for a Studio notebook, which isn't useful to Estimators. should these both be included in this notebook, or should I create a separate tutorial?